include/ring-core/target.h

```diff
@@ -42,9 +42,9 @@
 #elif !(defined(__ORDER_LITTLE_ENDIAN__) && (__BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__)) && \
       !(defined(__ORDER_BIG_ENDIAN__) && (__BYTE_ORDER__ == __ORDER_BIG_ENDIAN__))
 #error "Unsupported endianness"
-#elif defined(__LP64__)
+#elif defined(__LP64__) || defined(__loongarch64) || defined(__loongarch_lp64)
```

Comment:
Why is this needed? Does GCC for the 64-bit target not define `__LP64__`? https://loongson.github.io/LoongArch-Documentation/LoongArch-toolchain-conventions-EN.html says it is defined so this shouldn't be needed.